File: bubblesort/bubblesort.R
bubbleSort <- function(array) {
  n = length(array);
  for (i in 1:(n-1)) {
    print(array);
    for (j in 1:(n-i)) {
      if(array[j] > array[j+1]) {
        copy = array[j];
        array[j] = array[j+1];
        array[j+1] = copy;
      }
    }
  }
  print(array)
  return(array)
}

#bubbleSort(c(5,4,3,2,1))

File: bubblesort/bubblesort.js
let bubbleSort = (arr) => {
    let len = arr.length;
    for (let i = 0; i < len; i++) {
        console.log(arr);
        for (let j = 0; j < (len-i); j++) {
            if (arr[j] > arr[j+1]) { 
                let copy = arr[j];
                arr[j] = arr[j+1];
                arr[j+1] = copy;
            }
        }
    }
    return arr;
}

bubbleSort([30,25,45,60,8])